conda/recipes/librmm/meta.yaml

```
@@ -27,8 +27,10 @@ requirements:
     - cmake >=3.12.4
   host:
     - cudatoolkit {{ cuda_version }}.*
+    - spdlog >=1.4.2
   run:
     - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}
+    - spdlog >=1.4.2
```

Would suggest tweaking this a little bit. Should make it easier to bump the version as needed.

```suggestion
  host:
    - cudatoolkit {{ cuda_version }}.*
    - spdlog 1.4.2
  run:
    - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}
    - {{ pin_compatible('spdlog', max_pin='x.x') }}
```

---

Wouldn't this make it stuck at 1.4.2 until we update it? Anyway, spdlog is a header-only library and we are not using any fancy features, the versioning probably doesn't need to be that strict. Perhaps we just leave out the version altogether?